Plugins/Json/src/AmbiguityConfigJsonConverter.cpp

```diff
@@ -35,15 +35,26 @@ void from_json(const nlohmann::json& j, ConfigPair& p) {
 
     detectorConfig.sharedHitsFlag = value["sharedHitsFlag"];
 
-    std::vector<double> factorHits = value["factorHits"];
-    std::vector<double> factorHoles = value["factorHoles"];
+    std::vector<double> goodHits = value["goodHits"];
+    std::vector<double> goodHoles = value["goodHoles"];
 
-    for (auto factor : factorHits) {
-      detectorConfig.factorHits.push_back(factor);
+    std::vector<double> fakeHits = value["fakeHits"];
+    std::vector<double> fakeHoles = value["fakeHoles"];
```

Comment:
the question remains if these can be const references or if we really need copies.
Can this be made a 
```suggestion
    const std::vector<double>& fakeHits = value["fakeHits"];
    const std::vector<double>&fakeHoles = value["fakeHoles"];
```